### ISSUE TITLE
Task src

### ISSUE THREAD

**Issue by vilmosioo** [TypeStrong/grunt-ts#305]
When calling a task as "src", we get the false warning:

```
Warning: In task "src", either "files" or "src" should be used - not both.
```

Renaming the task to anything else, the warning disappears.

Configuration:
```
options: {
	fast: "never",
	noImplicitAny: true,
	failOnTypeErrors: true,
	suppressImplicitAnyIndexErrors: true,
	removeComments: false,
	target: "es5",
	sourceRoot: ""
},

default: {
	files: [{
		src: ["build/**/*.ts", "typings/**/*.d.ts"],
		dest: "build"
	}]
}
```
Version used: "5.1.0"

**Comment by nycdotnet** [comment]
The task itself is named src?  That's interesting.  I'll take a look.  Thanks for the report.

**Comment by nycdotnet** [comment]
This should be resolved with v5.1.1 via ab09c2e5b7e807a36b8ba2c5fff8d2df8b4e23b9.  Please re-open if not.  Thanks for the report.
